scs2-definition/src/main/java/us/ihmc/scs2/definition/robot/urdf/URDFTools.java

```diff
@@ -209,66 +247,256 @@ public static URDFModel loadURDFModel(InputStream inputStream, Collection<String
     *                            done.
     * @return the model.
     */
-   public static URDFModel loadURDFModel(InputStream inputStream,
+   public static URDFModel loadURDFModel(Collection<InputStream> inputStreams,
                                          Collection<String> resourceDirectories,
                                          ClassLoader resourceClassLoader,
                                          URDFParserProperties parserProperties) throws JAXBException
```

Comment:
Let's add another factory for backward compatibility with the signature:
`URDFModel loadURDFModel(InputStream inputStream, Collection<String> resourceDirectories, ClassLoader resourceClassLoader, URDFParserProperties parserProperties)`

Reply:
We have this... lets see if I can link it....

Reply:
Couldn't link it but its on line 210